docs/sources/set-up/install-agent-macos.md

```diff
@@ -33,8 +33,13 @@ Use Homebrew to install the most recent released version of Grafana using the Ho
     ```
 
 1. Modify `config.yml` with your configuration requirements. 
+    
+    ```
+    vi $(brew --prefix)/etc/grafana-agent/config.yml
```

Comment:
@ricardolupo I'm a little cautious that not everyone might be familiar with Vi here and might get stuck. 

WDYT about telling users where to find the config file in the previous line instead?